qemu/tests/cfg/image_commit_bypass_host_cache.cfg

```diff
@@ -14,7 +14,7 @@
     backup_image_before_testing = yes
     restore_image_after_testing = yes
     trace_events = open
-    Host_RHEL.m8, Host_RHEL.m9:
+    Host_RHEL.m8, Host_RHEL.m9, Host_RHEL.m10:
```

Comment:
Hi @hellohellenmao , I guess that we'll always use openat since RHEL8, so what about

trace_events = openat
Host_RHEL.m6, Host_RHEL.m7:
    trace_events = open
(we do have Host_RHEL.m5, Host_RHEL.m4 and Host_RHEL.m3, but I cannot imagine we'll run tests on these ancient releases, but  you can add them all with the explicitness )
Then we don't consider this when a new release comes out, what's your opinion?

Reply:
@zhencliu Yeah, good point. Agree with you, let me update it. Thanks.